src/main/java/org/kohsuke/github/authorization/OrgAppInstallationAuthorizationProvider.java

```diff
@@ -17,47 +23,75 @@
  */
 public class OrgAppInstallationAuthorizationProvider extends GitHub.DependentAuthorizationProvider {
 
-    private final String organizationName;
+    private static final Pattern pattern = Pattern.compile("/repos/(.*)/.*");
```

Comment:
This provider has a reference to a `GitHub` instance, meaning we can get the base URL using `getApiUrl()` and then do simpler string matching.  I'd rather not regex if we don't have to. 

Also do GitHub App tokens only apply to `/repos/*` endpoints  or can they interact with other endpoints?